src/processpriv.h

```diff
@@ -77,16 +77,16 @@ class ProcessPriv : public AbstractPrivateData {
     DLLLOCAL static void waitForTermination(int pid, ExceptionSink* xsink);
 
 private:
-    ResolvedCallReferenceNode* optsExecutor(const char* name, const QoreHashNode* opts, ExceptionSink* xsink);
+    DLLLOCAL ResolvedCallReferenceNode* optsExecutor(const char* name, const QoreHashNode* opts, ExceptionSink* xsink);
```

Comment:
btw instead of defining `DLLLOCAL` everywhere, we can use anonymous namespace for the whole classes, but I'm fine with this solution as well, so up to you.